packages/framework/src/Resources/views/Common/Inline/Icon/icon.html.twig

```diff
@@ -0,0 +1,8 @@
+{% if type == 'svg-font' %}
+    <i class="svg svg-{{ name }} {{ attr.class|default('')}}"
```

Comment:
What about to create some default attributes right in `renderIcon` method when it will be merged with given arguments to prevent using `|default('')` everywhere?

Reply:
changed

Reply:
I am missing a default value for `attr.data`

Reply:
added